File: ptt-gc-spa/src/app/core/models/customReportMerge.ts
import { CustomReportHeader } from './customReportHeader';
import { CustomReportDetail } from './customReportDetail';
import { CustomReportParam } from './customReportParam';

export interface CustomReportMerge {
    reportHeader: CustomReportHeader;
    reportDetailX: CustomReportDetail;
    reportDetailY: CustomReportDetail[];
    reportParam: CustomReportParam[];

}

File: ptt-gc-api/Dtos/SettingChart/CustomReportMerge.cs
using PTT_GC_API.Models.SettingChart;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PTT_GC_API.Dtos.SettingChart
{
    public class CustomReportMerge
    {
        public CustomReportHeader ReportHeader { get; set; }
        public CustomReportDetail ReportDetailX { get; set; }
        public CustomReportDetail[] ReportDetailY { get; set; }
        public CustomReportParameter[] ReportParam { get; set; }
    }
}
